deploy/runtime/ui/Fuel Cell O and M Costs.txt

```diff
@@ -2469,7 +2469,16 @@ equations{ 'add_om_num_types'} = define(){
 	return 2;
 };
 equations{ 'om_capacity1_nameplate'} = define(){
-	return ${batt_computed_bank_capacity};
+	if( technology() == 'PV Battery' || technology() == 'Fuel Cell')
+			return ?${en_batt}
```

Comment:
what about generic battery?

Reply:
@sjanzou This is on the Fuel Cell O and M Costs form that is only used for the Fuel Cell configurations, so no need for the configuration checks for other battery techs here.